components/whyd/2022/Terminal/UserInputArea.vue

```diff
@@ -230,18 +236,26 @@ export default {
       this.focusInput(false)
     },
     processCommand(input) {
-      const processed = input.trim() // .toLowerCase() fuck you if you try to use caps in the terminal
-
-      if (this.validCommands.has(processed)) {
-        const command =
-          this.terminalCommands[input] ??
-          this.terminalCommands[
-            Object.entries(this.terminalCommands).find(
-              (e) => e[1].prettyName === processed
-            )[0]
-          ]
-        command.hasBeenRun = true
-        // const command = this.terminalCommands[input]
+      let commandName = input.trim() // .toLowerCase() fuck you if you try to use caps in the terminal
+
+      if (this.validCommands.has(commandName)) {
+        let command = this.terminalCommands[commandName]
```

Comment:
`command` and `commandName` are now mutable so I could mutate them to the correct values in case the command has to be found via `prettyName`

Reply:
we'll talk about this